.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "deps/snappy/snappy"]
+	path = deps/snappy/snappy
+	url = https://github.com/google/snappy.git
```

Comment:
Can we specify a commit tag here to lock the version?
I remember that git submodule will always update to the latest commit of a branch. So future changes in snappy might break our code?

Reply:
It should already be fixed to the `1.1.7` tag. Can you try by checking out this commit?